Gems/EMotionFX/Code/Include/Integration/AnimationBus.h

```diff
@@ -169,17 +169,13 @@ namespace EMotionFX
 
 
         /**
-         * The raycast request bus, which EMotion FX calls in order to perform ray cast tests.
+         * The raycast request interface, which EMotion FX calls in order to perform ray cast tests.
          * This allows you to perform custom filtering.
          */
-        class RaycastRequests : public AZ::EBusTraits
+        class IRaycastRequests
         {
         public:
-            // Enable multi-threaded access by locking primitive using a mutex when connecting handlers to the EBus or executing events.
-            using MutexType = AZStd::recursive_mutex;
```

Comment:
Change looks good. Quick question before approving. Normally when the mutex is used in a bus is because is needed to be called from multiple threads and this protects the bus so the calls don't happen at the same time. Will the interface work appropriately if it's called from multiple threads?

Reply:
Calling from multiple threads is exactly when the performance issues from EBus locks arise. 
Yes, the raycast itself is thread safe.